clients/nodeRating/nodeRating.go

```diff
@@ -0,0 +1,164 @@
+package noderating
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/node-monitoring/clients"
+	"github.com/ElrondNetwork/node-monitoring/common"
+	"github.com/ElrondNetwork/node-monitoring/config"
+	"github.com/ElrondNetwork/node-monitoring/data"
+)
+
+var log = logger.GetOrCreate("clients/nodeRating")
+
+const (
+	firstRunValue    float64 = -1
```

Comment:
unused value. Is it still needed? Also: minReqTimeoutSec and nodesIdentifierPath

Reply:
forgot about them after refactoring, removed them